src/qiskit_sphinx_theme/assets/styles/qiskit-sphinx-theme.scss

```diff
@@ -11,6 +11,10 @@
  * that they have been altered from the originals.
  */
 
+// Import the base Furo style
+@import "furo";
```

Comment:
This is how we import Furo's style now.

The next file in this PR, `furo_py/__init__.py` is really important to review btw.